src/main/services/pre-api/pre-client.ts

```diff
@@ -14,6 +14,18 @@ export class PreClient {
     await axios.get('/health');
   }
 
+  public async putAudit(xUserId: string, request: PutAuditRequest): Promise<AxiosResponse | undefined> {
```

Comment:
You'll need to check this but you may be able to keep this return signature as `Promise<AxiosResponse>` and rethrow the exception after logging it? Or potentially remove the catch as I think it'll get caught and logged by the global handler anyway.